tensorflow/lite/experimental/micro/kernels/cmsis-nn/conv.cc

```diff
@@ -235,7 +236,15 @@ TfLiteStatus EvalQuantizedPerChannel(
     }
   }
 #else
-#error ARM_MATH_DSP and ARM_MATH_LOOPUNROLL must be set
+
```

Comment:
Actually, it would be good for the user to be notified that there's no optimization for this target. You can add a directive for that. For example:
#pragma message("CMSIS-NN optimization for conv not available for this target. Using ref kernel.")

Reply:
Great idea!

Reply:
I'm thinking it'd be better to have a more descriptive notification. i.e:

"CMSIS-NN optimization for _conv_ not available for this target (ARM_MATH_DSP and ARM_MATH_LOOPUNROLL not set). Using reference kernel"

what do you say?

Reply:
done!